CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(metalang99)
+
+add_library(${PROJECT_NAME} INTERFACE)
```

Comment:
```suggestion
add_library(Metalang99 INTERFACE)
```

Reply:
This is just simpler (had to google what `${PROJECT_NAME}` means, is this the top project in the build hierarchy or not, etc.)

Reply:
I used to do this for the same reason, but ${PROJECT_NAME} saves a lot of pain if you ever want to change the project name for some reason or in the case where you want to copy snippets between projects.

Reply:
Makes sense.